frontend/i18next.scanner/en/translation.json

```diff
@@ -425,5 +425,6 @@
   "Validate anyway": "Validate anyway",
   "Simple edition": "Simple edition",
   "Advanced edition": "Advanced edition",
-  "Publish": "Publish"
+  "Publish": "Publish",
+  "Publication of {{author}} at {{date}}": "Publication of {{author}} at {{date}}"
```

Comment:
I think "on" is more appropriate than "at" before an absolute date.

see for instance https://www.englisch-hilfen.de/en/words/date.htm (look for "complete date")